input/data/2017/2017A%26A...603A..31A/tev-000091-1.yaml

```diff
@@ -0,0 +1,15 @@
+source_id: 91
+file_id: 1
+reference_id: 2017A%26A...603A..31A
+telescope: veritas
+
+# VERITAS high (TeV flare  2009 May 1)
+spec:
+  erange: {min: 0.2, max: 6., unit: TeV}
+  mjd: {max: 54955., min: 54952.41}
+  model:
+    type: pl
+    parameters:
+      e_ref: {val: 1., unit: TeV}
+      norm: {val: 4.17, err: 0.24, scale: 1e-11, unit: cm-2 s-1}
```

Comment:
This should be in unit `cm-2 s-1 TeV-1`.
(I checked the paper: https://arxiv.org/pdf/1612.09472.pdf)